wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandScheduler.java

```diff
@@ -86,7 +86,8 @@ public static synchronized CommandScheduler getInstance() {
   // scheduled/canceled during run
   private boolean m_inRunLoop;
   private final Set<Command> m_toSchedule = new LinkedHashSet<>();
-  private final List<Command> m_toCancel = new ArrayList<>();
+  private final Set<Command> m_toCancel = new LinkedHashSet<>();
```

Comment:
Rationale for this change? We want to maintain insertion order.

Reply:
I think I just changed it when I switched to sets instead of queues.